appium.js

```diff
@@ -1,15 +1,32 @@
 /* eslint-disable no-console */
 
-import { ipcMain } from 'electron';
+import { ipcMain, BrowserWindow, Menu } from 'electron';
 import { main as appiumServer } from 'appium';
 import { getDefaultArgs, getParser } from 'appium/build/lib/parser';
+import path from 'path';
+import wd from 'wd';
 
 const LOG_SEND_INTERVAL_MS = 250;
+const isDev = process.env.NODE_ENV === 'development';
 
 var server = null;
+var serverArgs = null;
 var logWatcher = null;
 var batchedLogs = [];
 
+let sessionDrivers = {};
+
+/**
+ * Kill session associated with session browser window
+ */
+function killSession (sessionWinID) {
+  if (sessionDrivers[sessionWinID]) {
+    sessionDrivers[sessionWinID].closeApp();
```

Comment:
instead of calling `closeApp` and `end`, we should be calling `quit`. and it should be `await`ed